LearningMachine/app/build.gradle

```diff
@@ -136,19 +136,16 @@ final ESPRESSO_VERSION = '3.0.1'
 // test
 final JUNIT_VERSION = '4.12'
 final MOCKITO_VERSION = '2.2.11'
-final ROBOLECTRIC_VERSION = '4.4'
+final ROBOLECTRIC_VERSION = '4.14.1'
 final JODATIME_VERSION = '2.8.1'
 
 // These must be forced until everyone gets on the same page
-// ?: 18.0, Robolectric: 20.0
+// ?: 18.0, Robolectric: 33.3.1-jre
 final GUAVA_VERSION = '27.0.1-android'
-// RxLifecycle: 3.0.1, Robolectric: 2.0.1
```

Comment:
Robolectric now uses FindBugs 3.0.2, so I took the liberty of removing this. Let me know if you prefer to keep it.